frontend/micro-ui/web/micro-ui-internals/packages/react-components/src/hoc/FormComposer.js

```diff
@@ -281,7 +281,7 @@ export const FormComposer = (props) => {
   };
   return (
     <form onSubmit={handleSubmit(onSubmit)} onKeyDown={(e) => checkKeyDown(e)} id={props.formId} className={props.className}>
-      <Card style={getCardStyles()} className={props?.className}>
+      <Card style={getCardStyles()} className={props?.className} {...props}>
```

Comment:
@prachirane-eGov Can you check this change? Is it required?

Reply:
@nipunarora-eGov 
Yes, it is required. Since we are using Formcomposer here, card inside it by default takes particular class if we are logging in as Employee but those style changes are not required in this module so to avoid that I needed to pass 'noCardStyle' prop to Card component

Reply:
You can just send 'noCardStyle' prop if it is there in list of props instead of sending all of them